python/cudf/cudf/tests/test_numba_import.py

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.
+import subprocess
+import sys
+
+import pytest
+
+IS_CUDA_11 = False
+try:
+    from ptxcompiler.patch import NO_DRIVER, safe_get_versions
+
+    versions = safe_get_versions()
+    if versions != NO_DRIVER:
+        driver_version, runtime_version = versions
+        if driver_version < (12, 0):
+            IS_CUDA_11 = True
+except ModuleNotFoundError:
+    pass
+
+TEST_NUMBA_MVC_ENABLED = """
+import numba.cuda
+import cudf
+from cudf.utils._numba import _CUDFNumbaConfig, _patch_numba_mvc
+
+from numba import config
+config.CUDA_ENABLE_MINOR_VERSION_COMPATIBILITY=1
```

Comment:
I think we decided this wasn't necessary?
```suggestion
```

Reply:
Correct. Committing this change and adding `/merge`.